src/driver/sqlserver/SqlServerDriver.ts

```diff
@@ -891,7 +891,11 @@ export class SqlServerDriver implements Driver {
     /**
      * Returns true if driver supports RETURNING / OUTPUT statement.
      */
-    isReturningSqlSupported(): boolean {
+    isReturningSqlSupported(returningType: ReturningType): boolean {
+        if (returningType === 'insert') {
+          return false;
```

Comment:
It was disabled in [query-builder/InsertQueryBuilder.ts](https://github.com/typeorm/typeorm/pull/10760/files#diff-f12ef5527b9bc2971ab7a8247c71dd735a43cd7a1c859bd206b61dee8c7f617d)